### PR TITLE
Process to transfer Google docs into the shared Google Drive

### PR DESCRIPTION
**Pull request by chensun**
Related to #308

/assign @jlewi

### CODE REVIEW

**Comment by chensun** [comment]
/cc @jlewi 
Sorry for the long delay of putting up this PR.

File: how-to/sharing_docs.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+You should be a member of the [kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss) group in order to view docs in the shared Google Drive.
+
+## Moving Google Docs into the shared Google Drive
+
+1. Assign ownership of the Google doc to drive-content-managers@kubeflow.org ([instruction](https://support.google.com/drive/answer/2494892)).
+1. Open a GitHub issue to transfer the doc.
+1. Assign the issue to someone in the [content manager list](#shared-google-drive-content-managers).
+1. Content manager will then move the doc into the drive.
+
+## Shared Google Drive Content Managers:
```
Comment (chensun) [inline how-to/sharing_docs.md:14]: This is the list of people with @google.com email in the drive-content-managers file in https://github.com/kubeflow/internal-acls. I'm not able to find the GitHub usernames for those with only @kubeflow.org email.

**Comment by jlewi** [comment]
Thanks

/lgtm
/approve

File: how-to/sharing_docs.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Sharing Docs with the Community
+
+## Prerequisites
+
+You should be a member of the [kubeflow-discuss](https://groups.google.com/forum/#!forum/kubeflow-discuss) group in order to view docs in the shared Google Drive.
+
+## Moving Google Docs into the shared Google Drive
+
+1. Assign ownership of the Google doc to drive-content-managers@kubeflow.org ([instruction](https://support.google.com/drive/answer/2494892)).
```
Comment (chensun) [inline how-to/sharing_docs.md:9]: @jlewi I just realized that this instruction doesn't work, I cannot assign ownership to this group email regardless if it's a doc owned by an @google.com account or an @gmail.com account. The "Make owner" option is always grayed out.
Furthermore, it seems Google doc doesn't even allow transfer ownership to individual email address if the email belongs to a different domain.
Would you give another thought on the process? Thanks!

**Pull request merged**